test/jdk/java/awt/Focus/InactiveFocusRace.java

```diff
@@ -97,18 +106,12 @@ public void windowActivated(WindowEvent e) {
                 sema.raise();
             }
         });
+        inactiveFrame.setVisible(true);
```

Comment:
The previous version used a Semaphore to wait until the frame became active or the button received focus, but it didn’t check the second inactive frame.
I think what we should do is first show the "inactive" frame using the same Semaphore. Then, once it becomes visible, we should show another "active" frame.
This will ensure that the frame we intend to click below remains inactive/unfocused.

Reply:
I didn't get the "previous version"..I just added a delay between the UI creation and test start.
Earlier also, init is called to do UI creation and then start is immediately called which sets 
```
inactiveFrame.setVisible(true);
        activeFrame.setVisible(true);
```
All the semaphore logic is still the same after that